code/_onclick/hud/ai.dm

```diff
@@ -142,6 +146,8 @@
 	icon_state = "view_images"
 
 /obj/screen/ai/image_view/Click()
+	if(..())
+		return
 	if(isAI(usr))
```

Comment:
`if(!..() && isAI(usr))`
and remove the
`if(..()) return`

I think you can do the same for line 137 too.

Reply:
There's a second check for iscyborg() and I don't see any reason to allow cyborgs to take pictures while incapacitated.

Reply:
K